wpilibj/src/main/java/edu/wpi/first/wpilibj/Ultrasonic.java

```diff
@@ -116,8 +116,8 @@ public int getEchoChannel() {
   }
 
   /**
-   * Create an instance of the Ultrasonic Sensor. This is designed to supchannel the Daventech SRF04
-   * and Vex ultrasonic sensors.
+   * Create an instance of the Ultrasonic Sensor. This is designed to sub channel the Daventech
```

Comment:
This should read “support”—it was a search and replace oops when changing from “port” to “channel”.